src/PhpSpec/Process/ReRunner/PassthruReRunner.php

```diff
@@ -43,7 +43,8 @@ public function isSupported()
     {
         return (php_sapi_name() == 'cli')
             && $this->getExecutablePath()
-            && function_exists('passthru');
+            && function_exists('passthru')
+            && strtolower(substr(PHP_OS, 0, 3)) != "win";
```

Comment:
I've had a search through the codebase but can't find any other places you are detecting windows any ideas where it might be?